Projekty/Projekt1/Kacper_Grzymkowski_Jan_Gaska/Eksploracja_danych/WUM_Kacper_Grzymkowski_Jan_Gaska_Eksploracja.ipynb

```diff
@@ -0,0 +1,1569 @@
+{
```

Comment:
<p>To, że więcej demokratów wstrzymuje się od głosu może wynikać też z tego, że partia Demokratów jest większa. Teoretycznie mogłoby się okazać, że proporcja osób wstrzymujących się w danej partii do pozostałych jej członków jest zbliżona w obu partiach. To jest niuans, o którym należy pamiętać w przypadku niezbalansowanych zbiorów i porównywania liczb bezwzględnych.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>